models/reviewsModel.js

```diff
@@ -1,19 +1,62 @@
 const db = require("../db/connection.js");
 
-const fetchReviews = () => {
-  return db
-    .query(
-      `SELECT reviews.*, 
-      COUNT(comment_id) AS comment_count 
-      FROM reviews 
-      LEFT JOIN comments 
-      ON comments.review_id = reviews.review_id 
-      GROUP BY reviews.review_id 
-      ORDER BY created_at DESC;`
-    )
-    .then(({ rows: reviews }) => {
-      return reviews;
+const fetchReviews = (sortBy = "created_at", orderBy = "DESC", category) => {
+  const queryValues = [];
+
+  if (category) {
+    category = category.toLowerCase();
+  }
+
+  const validOrderOptions = ["ASC", "DESC"];
+  const validCategories = ["euro game", "social deduction", "dexterity"];
```

Comment:
We can use SQL paramters in SQL so don't need to do a check like we do for the sortBy and OrderBy queries.  This will effect your code below here too.